src/esp/sim/Simulator.cpp

```diff
@@ -40,6 +43,14 @@ using metadata::attributes::SceneAOInstanceAttributes;
 using metadata::attributes::SceneObjectInstanceAttributes;
 using metadata::attributes::StageAttributes;
 
+namespace {
+constexpr const char* shadowMapDrawableGroupName = "static-shadow-map";
+constexpr const char* defaultRenderingGroupName = "";
+const int shadowMapSize = 1024;
+const int maxNumShadowMaps = 3;  // the max number of point shadow maps
```

Comment:
```suggestion
constexpr int shadowMapSize = 1024;
constexpr int maxNumShadowMaps = 3;  // the max number of point shadow maps
```

Re naming, just FYI -- in Magnum I'm having constants with an uppercase first letter (treating them the same as enums, basically). 

Not a fan of `k` or `g` either :)